index.js

```diff
@@ -0,0 +1,105 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+fs = require('fs');
+
+var octokit;
+
+const orgName = core.getInput('org-name');
+const authToken = process.env["GITHUB_TOKEN"];
+const path = core.getInput("file-path");
+const name = core.getInput("commit-user-name");
+const email = core.getInput("commit-user-email");
+const message = core.getInput("commit-msg")
+// Eliminate Owner name from repo;
+const repo = process.env["GITHUB_REPOSITORY"].split("/")[1]
+
+/*
+// For Testing Locally
+
+function get_test_values(){
+    return JSON.parse(fs.readFileSync('test_data.json', { encoding: 'utf8' }));
+}
+var test_data = get_test_values();
+const orgName = test_data["org_name"]
+const authToken = test_data["token"];
+const path = test_data["path"]
+const name = test_data["name"]
+const email = test_data["email"]
+const message = test_data["message"]
+const repo = test_data["repo"]
+*/
+
+
+async function getMemberData(teams){
+    // TODO: find async forEach alternative
+    member_data = {};
+    for(var i=0; i < teams.length; i++){
+        var resp = await octokit.teams.listMembersInOrg({
+            org: orgName,
+            team_slug: teams[i].slug,
+        });
+        member_data[teams[i].name] = resp.data
+    }
+    return member_data;
+}
+
+async function run(){
+    try{       
+        if (!authToken) {
+            throw new Error("Token not found");
+        }
+
+        octokit = github.getOctokit(authToken);
+
+        teams_response =  await octokit.teams.list({
+            org: orgName,
+        }).catch(
+            err => console.log(err)
+        );
+        
+        member_data = await getMemberData(teams_response.data);
+        
+        json_file_response = await octokit.repos.getContent({
+            owner: orgName,
+            repo: repo,
+            path: path,
+        }).catch(
+            err => console.log(err)
```

Comment:
You can exit the app here if the file API goes wrong. Print error using `core.error(e)`

Reply:
@UKnowWhoIm and this

Reply:
I changes this too [index.js](https://github.com/UKnowWhoIm/members/blob/main/index.js) @ line 78